tests/testthat/test-help-handlers.R

```diff
@@ -26,7 +26,7 @@ Just a placeholder here.
 "
 
   doctree <- sphinx_doctree_from_doc(docs)
-  expect_equal(names(doctree$ids), c("parameters", "returns", "section1"))
+  expect_setequal(names(doctree$ids), c("parameters", "returns", "section1"))
```

Comment:
It seems like the ordering of names is not consistent, so using `setequal()` here seemed more appropriate.